.github/workflows/ci.yml

```diff
@@ -0,0 +1,136 @@
+name: CI/CD
+
+on: [push, pull_request]
```

Comment:
Maybe
```
on:
  push:
    branches:
      - master
  pull_request:
```

Reply:
Thanks for the review! This would reduce redundant testing... let me think about this a bit but I’m leaning toward this change.

Reply:
I did end up changing this. Thanks!